docs/docsite/rst/user_guide/playbooks_filters_ipaddr.rst

```diff
@@ -474,10 +478,10 @@ smaller subnets::
     # {{ address | ipsubnet(18, -5) }}
     192.168.144.0/27
 
-By specifying an other subnet as a second argument, if the second subnet include
-the first you can have the rank of the first subnet in the second ::
+By specifying another subnet as a second argument, if the second subnet includes
+the first, you can determine the rank of the first subnet in the second ::
 
-    # The rank of the ip in the subnet (the ip is the 36870nth /32 of the subnet)
+    # The rank of the IP in the subnet (the important is the 36870nth /32 of the subnet)
```

Comment:
I think autocorrect got a little trigger-happy here, changing `ip` to `important`